lib/screens/new_game1/game_name_field.dart

```diff
@@ -0,0 +1,29 @@
+import 'package:flutter/material.dart';
+import 'package:niira/models/view_models/new_game1.dart';
+import 'package:niira/services/game_service.dart';
+import 'package:niira/utilities/validators.dart' as validators;
+import 'package:provider/provider.dart';
+
+class GameNameField extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    return Padding(
+      padding: const EdgeInsets.fromLTRB(30, 20, 30, 0),
+      child: TextFormField(
+        key: Key('new_game1_name_field'),
+        decoration: InputDecoration(
+          labelText: 'Name',
+          focusedBorder: OutlineInputBorder(
+            borderSide: BorderSide(color: Colors.black, width: 2.0),
+          ),
+          enabledBorder: OutlineInputBorder(
+            borderSide: BorderSide(color: Colors.black, width: 2.0),
+          ),
+        ),
+        validator: validators.oneToFifteenChars,
+        onChanged: (val) =>
+            context.read<GameService>()?.newGameViewModel1?.name = val,
```

Comment:
@tamari-gray you know what I still don't get - is why we need the null checks... 

the test broke without them but why is the vm or  the game service ever null? 

anyway not worth holding up  the PR imo, just wanted to see if you had any thoughts

Reply:
Oh maybe there's an onChanged event in the first build of the widget tree, before everything is set up?

Reply:
I think I figured it out, we're using a mock service without all the things... got a fix a brewing...